wagtail_ab_testing/models.py

```diff
@@ -216,6 +216,30 @@ def check_for_winner(self):
             else:
                 return self.Variant.TREATMENT
 
+    def get_status_description(self):
+        """
+        Returns a string that describes the status in more detail.
+        """
+        status = self.get_status_display()
+
+        if self.status == AbTest.Status.RUNNING:
+            participants = self.hourly_logs.aggregate(participants=Sum('participants'))['participants'] or 0
+            completeness_percentange = int((participants * 100) / self.sample_size)
```

Comment:
What if someone sets the sample_size to zero? We may want to catch a ZeroDivision error or prevent a sample_size from ever being zero.

Reply:
Added a ``MinValueValidator`` to the field now which should make sure this isn't zero